docs/site/content/community/_index.html

```diff
@@ -27,7 +27,7 @@ <h3><a href="https://github.com/vmware-tanzu/community-edition">Dig into the cod
                 <img src="/img/illustrations/join-the-conversation.jpg" height="100" alt="" />
             </div>
             <div class="content">
-                <h3><a href="https://kubernetes.slack.com/archives/C02GY94A8KT">Join the conversation</a></h3>
+                <h3><a href="{{ .Site.Params.slack_url }}">Join the conversation</a></h3>
```

Comment:
Hmmm should this be the actual link?

When clicking this, I get
```
https://deploy-preview-2130--quirky-franklin-8969be.netlify.app/community/%7B%7B%20.Site.Params.slack_url%20%7D%7D
```

which results in a 404

Reply:
Yeah. That's not right. Let me see where I made a mistake.

Reply:
So that value is set here: https://github.com/vmware-tanzu/community-edition/blob/main/docs/site/config.yaml
And successfully referenced in our footer: https://github.com/vmware-tanzu/community-edition/blob/main/docs/site/themes/template/layouts/partials/footer.html

@jonasrosland please advise.

Reply:
I'll have another look tomorrow and close if it's unnecessary.